### ISSUE TITLE
[uBO Legacy RC] Logger problem

### ISSUE THREAD

**Issue by kowith337** [gorhill/uBlock#3464]
### Describe the issue
The logger is show everything from everywhere, no matter what tab you selected from drop-down menu

### One or more specific URLs where the issue occurs
- uBlock Logger
- Any URLs in another tab/window for test case

### Screenshot in which the issue can be seen
https://github.com/gorhill/uBlock/issues/3464#issuecomment-360815325
![Waterfox](https://user-images.githubusercontent.com/16535467/35442485-02dda9c8-02da-11e8-86e6-b6b819c363d8.png)
![Cyberfox](https://user-images.githubusercontent.com/16535467/35442244-227ac46a-02d9-11e8-8635-97543a237445.png)
![Pale Moon](https://user-images.githubusercontent.com/16535467/35442037-536c78c6-02d8-11e8-8369-a4c5e476528c.png)

### Steps for anyone to reproduce the issue
1. Make sure you're still use any **pre-quantum Firefox**
> - Firefox ESR
> - Pale Moon
> - Cyberfox
> - ~~Waterfox~~
> - something else...
2. Make sure the `xpinstall.signatures.required` set to `false` in **about:config** to let you able install legacy version of uBO from [here](https://github.com/gorhill/ublock/releases).
3. Install newest uBO pre-release legacy version (uBlock0-firefox.xpi)
4. Open Logger and select ~~**Current Tab** or~~ something else that not related to tab/window that you're testing (e.g. Blank Page, Start Page, Behind the scene, etc.)
5. Start navigate in new tab/window and see the logger
6. ~~If you didn't pick the **Behind the scene** from step 4, try to clear logs (eraser button) and you can see the BTS log entries will not removed~~

### Your settings
- OS/version: Windows 10 Pro Insider 64-bit
- Browser/version
  - Pale Moon 27.7.1
  - Firefox ESR 52.6.0
  - Cyberfox 52.6.1
  - ~~Waterfox 56.0.3~~
- uBlock Origin version: 1.14.25rc1

### Your filter lists
Out-of-box default

### Your custom filters (if any)
Out-of-box default

**Comment by uBlock-user** [comment]
Gorhill recently said that the logger has been unified, so I suppose this is what he meant by that.

**Comment by gorhill** [comment]
By design: "Current tab" will show network requests from current tab and those from behind-the-scene.

**Comment by kowith337** [comment]
Hmm, sorry for that.
But here's another test case, I've choose `about:addons` to observe, then I've refresh the your uBlock releases page and then...

In Pale Moon, it's still show everything, regardless of what tab you want to observe.
![image](https://user-images.githubusercontent.com/16535467/35446264-c16497ce-02e6-11e8-935e-814f8f07e75e.png)

Cyberfox also seems like happen as Pale moon...
![image](https://user-images.githubusercontent.com/16535467/35446586-c31ef27a-02e7-11e8-96da-7bcb9da7516e.png)

Waterfox is only throw BTS entry...
![image](https://user-images.githubusercontent.com/16535467/35446703-2726565a-02e8-11e8-9f40-484ae4551973.png)

**Comment by joey04** [comment]
I'd noticed this same problem when I tested the RC a few days ago.

FYI -- This weekend I published a forked uBO for Advanced user mode on legacy XUL browsers. It's based on v1.14.16, so it doesn't have these regressions.

https://github.com/joey04/xulBlock-Advanced

**Comment by anvakl** [comment]
Change 
```
sheet.insertRule(
            '#netInspector tr:not(.' + tabClass + '):not(.tab_bts) ' +
            '{display:none;}'
        );
```
in logger-ui.js to
```
sheet.insertRule(
            '#netInspector tr:not(.' + tabClass + '):not(.tab_bts) ' +
            '{display:none;}', 0
        );
```

**Comment by gorhill** [comment]
Fixed with pull request #3616.
